site/content/contribute/getting-started/code-review.md

```diff
@@ -107,3 +110,6 @@ If you are a core committer asked to give a review
     * Some changes are spread out across multiple pull requests that should be merged at the same time. Look out for the `Do Not Merge/Awaiting PR` label. When in doubt, leave the merging of the pull request to the author.
 5. Handle any cherry-picks.
     * There is an automated cherry-pick process and the author of the pull request should make sure the cherry-pick succeeds. Assume this is the case unless you are explicitly asked to help cherry-pick.
+6. After a pull request is merged (and cherry-picked where needed), update the Jira ticket.
```

Comment:
Do we want the reviewers to close the JIRA ticket? I'm concerned about reviewers closing JIRA's to quickly.

Thoughts?

Reply:
My concerns is that I've seen cases where the Jira ticket was never updated if another dev merged the PR, or the Jira ticket was updated multiple days after the PR was merged (I'm guessing at the next sprint meeting or such). Tickets should be resolved quickly so that they are ready for QA testing soon.

If we don't want other devs to update Jira tickets, then we need to make sure the submitter is doing that as soon as the PR is merged.

Reply:
Maybe we should dis encourage devs to merge other peoples PR (1/5) This way only the PR submitter closed his/her own PR and ensures that the JIRA also gets closed.

Site note: https://github.com/mattermost/mattermost-plugin-jira/issues/342 will help with ensuring JIRAs get closed in the future.

Reply:
> Maybe we should dis encourage devs to merge other peoples PR

One consideration I would add is community PRs - I've seen instances where their associated Jira tickets won't get resolved until later in the release cycle

Reply:
Community PRs are a bit different to me process wise. They don't get merged by the submitter and instead by a staff member. Which means the staff member who merged it has to also close the JIRA.

Reply:
Ok, removed step 6 here.